docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc

```diff
@@ -1,176 +1,77 @@
 [[running-on-kubernetes-managed-by-fleet]]
 = Run {agent} on Kubernetes managed by {fleet}
 
```

Comment:
I agree. Let's follow the structure we have at https://github.com/elastic/observability-docs/pull/2335 and https://github.com/elastic/observability-docs/pull/2360

Reply:
I changed the order. Since the GKE and EKS pages state kube state metrics as a pre requisite, I removed its deployment as a step. I also removed all the steps from those pages as they said the same thing as running-on-kubernetes-managed-by-fleet.asciidoc, and added a new line pointing to that page. I left the unique parts as they were.

Reply:
Thank you @constanca-m big improvement indeed. So I review this last commit https://github.com/elastic/observability-docs/blob/4f97593c515df8955294c25fcf249139836071a1/docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc
Small improvements:

- Step 1: Download the {agent} manifest should go before Step 5. Deploy Agent
- Added some suggestions as well below
- Also `== Step 4: Configure tolerations` can not be a seperate step? Can be part of Manifest step?

Reply:
1. Since the steps 3 and 4 mention settings and configurations of the manifest file, it seems confusing to give the reader a way to reach it only after we have mentioned those. What do you think @gizas?
2. Liked the suggestions, committed.
3. I guess it could, but it doesn't really relate with the previous steps: settings and agent policy. And this way we also can keep the same structure we used for agent container and standalone agent. But if this doesn't convince you, I can switch the order.